### ISSUE TITLE
terraform apply on in-place update of phone numbers result in a no-op

### ISSUE THREAD

**Issue by peterzcc** [PagerDuty/terraform-provider-pagerduty#583]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
1.1.0

### Affected Resource(s)
Please list the resources as a list, for example:
- pagerduty_user_contact_method

### terraform diff example
```

Terraform used the selected providers to generate the following execution
plan. Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # module.users.pagerduty_user_contact_method.all["xxxx,phone,GS Web"] will be updated in-place
  ~ resource "pagerduty_user_contact_method" "all" {
      ~ address          = "11111111" -> "22222222"
        id               = "xxxxxx"
        # (7 unchanged attributes hidden)
    }

  # module.users.pagerduty_user_contact_method.all["xxxx,sms,GS Web"] will be updated in-place
  ~ resource "pagerduty_user_contact_method" "all" {
      ~ address          = "11111111" -> "22222222"
        id               = "xxxx"
        # (7 unchanged attributes hidden)
    }

Plan: 0 to add, 2 to change, 0 to destroy.

Warning: Value for undeclared variable

The root module does not declare a variable named "svc_now_password" but a
value was found in file "/tfvars/ro-secrets.auto.tfvars". If you meant to use
this value, add a "variable" block to the configuration.

To silence these warnings, use TF_VAR_... environment variables to provide
certain "global" settings to all configurations in your organization. To
reduce the verbosity of these warnings, use the -compact-warnings option.

─────────────────────────────────────────────────────────────────────────────

Note: You didn't use the -out option to save this plan, so Terraform can't
guarantee to take exactly these actions if you run "terraform apply" now.

```

### Expected Behavior
The target phone number is enforced

### Actual Behavior
no-op and the diff persists

**Comment by gsreynolds** [comment]
Hi @peterzcc, which version of the Terraform provider are you using? 2.6.3 is the latest.

I wasn't able to reproduce the issue you are seeing on 2.5.0 or 2.6.3.

Plan:
```
  # pagerduty_user_contact_method.phone["cloud1"] will be updated in-place
  ~ resource "pagerduty_user_contact_method" "phone" {
      ~ address          = "5555551235" -> "5555551234"
        id               = "PXXXXXX"
        # (7 unchanged attributes hidden)
    }

  # pagerduty_user_contact_method.sms["cloud1"] will be updated in-place
  ~ resource "pagerduty_user_contact_method" "sms" {
      ~ address          = "5555551235" -> "5555551234"
        id               = "PXXXXXX"
        # (7 unchanged attributes hidden)
    }
```

Apply:
```
pagerduty_user_contact_method.phone["cloud1"]: Modifying... [id=PXXXXXX]
pagerduty_user_contact_method.sms["cloud1"]: Modifying... [id=PXXXXXX]
pagerduty_user_contact_method.phone["cloud1"]: Modifications complete after 2s [id=PXXXXXX]
pagerduty_user_contact_method.sms["cloud1"]: Modifications complete after 2s [id=PXXXXXX]

Apply complete! Resources: 0 added, 2 changed, 0 destroyed.
```

Plan again:
```
No changes. Your infrastructure matches the configuration.

Terraform has compared your real infrastructure against your configuration and found no differences, so no changes are needed
```

**Comment by imjaroiswebdev** [comment]
Hey @peterzcc the solution to this issue was addressed by #586 and it's available now to from the provider version **v2.6.5**. So I'm proceeding to close this issue, because it's hopefully solve. If you find are still facing the issue or an edge case not covered is affecting, please don't hesitate on reopening this issue thread ✌🏽 🙂
